.github/workflows/integrations.yml

```diff
@@ -0,0 +1,53 @@
+
+name: RMT Integration tests
+
+on: [pull_request]
+
+jobs:
+  test:
+    env:
+      DANGER_GITHUB_API_TOKEN: ${ secrets.GH_TOKEN }
+    services:
+      mysql:
+        image: mysql:latest
+        ports:
+          - 3306:3306
+        env:
+          MYSQL_DATABASE: rmt_test
+          MYSQL_USER: rmt
+          MYSQL_PASSWORD: rmt
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby-version: ['2.5.8', '2.6.6', '2.7.2']
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Ruby
+      uses: ruby/setup-ruby@473e4d8fe5dd94ee328fdfca9f8c9c7afc9dae5e
+      with:
+        ruby-version: ${{ matrix.ruby-version }}
+        bundler-cache: true
+        bundler: 1.17
+    - name: Gem cache
+      uses: actions/cache@v1
+      with:
+        path: vendor/bundle
+        key: ${{ matrix.ruby-version }}-gems-${{ hashFiles('Gemfile') }}
```

Comment:
From their documentation, you should be using the lockfile instead. Thus: `key: ${{ runner.os }}-gems-${{ hashFiles('**/Gemfile.lock') }}` (notice that the same cache should be used throughout the different ruby versions, so there's no need to keep a cache per ruby version, and it would kind of defeat its own purpose)